pkg/cfn/builder/cluster_test.go

```diff
@@ -360,6 +361,46 @@ var _ = Describe("Cluster Template Builder", func() {
 				Expect(clusterTemplate.Resources).ToNot(HaveKey("PublicSubnetRoute"))
 				Expect(clusterTemplate.Resources).To(HaveKey(ContainSubstring("PrivateRouteTable")))
 			})
+			When("ipv6 cluster is enabled", func() {
```

Comment:
doesn't this already exist in this file :thinking:

Reply:
Sh*t. I thought I moved this test case there already....

Reply:
Oh Actually! This is the important bit:

```go
		Context("when private networking is set", func() {
			BeforeEach(func() {
				cfg.PrivateCluster = &api.PrivateCluster{Enabled: true}
```

So this test case is in the section where private cluster is enabled. :)

Reply:
Because in that case there are some mock function calls which need to happen and those are all set up in this section. So that tops the IPV6 setting.

Reply:
mhmmmmmmmmm so you've gone for grouping `private clusters` instead of grouping `ipv6`. Thats fair :wink:

Reply:
![image](https://user-images.githubusercontent.com/19669095/140718868-9f34c1c8-357c-4801-a9aa-f31fa73cbcc7.png)